modules/gitops-release-notes-1-8-5.adoc

```diff
@@ -0,0 +1,37 @@
+/ Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-8-5_{context}"]
+= Release notes for {gitops-title} 1.8.5
+
+{gitops-title} 1.8.5 is now available on {product-title} 4.10, 4.11, 4.12, and 4.13.
+
+[id="errata-updates-1-8-5_{context}"]
+== Errata updates
+
+[id="rhsa-2023-5030-gitops-1-8-5-security-update-advisory_{context}"]
+=== RHSA-2023:5030 - {gitops-title} 1.8.5 security update advisory
+
+Issued: 2023-09-08
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2023:5030[RHSA-2023:5030]
+
+If you have installed the {gitops-title} Operator, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-operators
+----
+
+[id="fixed-issues-1-8-5_{context}"]
+== Fixed issues
+
+The following issue has been resolved in the current release:
+
+* Before this update, there was a mismatch in the RSA key for known hosts in the `argocd-ssh-known-hosts-cm` config map. This update fixes the issue by matching the RSA key with the upstream project. Now, you can use the default RSA keys on default deployments. link:https://issues.redhat.com/browse/GITOPS-3248[GITOPS-3248]
```

Comment:
Hi @masauso-lungu, this is fixed as part of 1.8.4
https://64187--docspreview.netlify.app/openshift-enterprise/latest/cicd/gitops/gitops-release-notes#gitops-release-notes-1-8-4_gitops-release-notes

@iam-veeramalla please confirm

Reply:
I'm also a little confused what's happening here... I do see that there was another customer issue opened for this that did affect 1.8.4, and a fix for it supposedly went into 1.8.5, but I don't see a pull request for it anywhere? Abhishek just added the RN text for it last week though so I think it's accurate, I just don't see what the change was and how it's different from the change in 1.8.4...

Reply:
I tried to dig into this more, I think that including this is a mistake. I think that it was indeed fixed in 1.8.4 but the customer issues opened against 1.8.4 were happening because of a cached error before upgrading. The case was closed and there was no mention of any changes made to address it. Sorry about the confusion @masauso-lungu, I think you can delete this part.

Reply:
Yes it is fixed in 1.8.4, @masauso-lungu please remove this.